src/core/evaluator.js

```diff
@@ -584,7 +584,8 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
       var glyphs = font.charsToGlyphs(chars);
       var isAddToPathSet = !!(state.textRenderingMode &
                               TextRenderingMode.ADD_TO_PATH_FLAG);
-      if (font.data && (isAddToPathSet || this.options.disableFontFace)) {
+      if (font.data && (isAddToPathSet || this.options.disableFontFace ||
+          state.fillColorSpace.name === 'Pattern')) {
```

Comment:
Nit: To make it easier to interpret this condition, can you please re-align it as follows instead?
```javascript
if (font.data && (isAddToPathSet || this.options.disableFontFace ||
                  state.fillColorSpace.name === 'Pattern')) {
```